native-extension/klarna/checkout/projects/klarna-checkout/src/lib/klarna-checkout/klarna-checkout.component.ts

```diff
@@ -88,19 +88,39 @@ export class KlarnaCheckoutComponent implements OnInit {
       )
       .pipe(
         concatMapTo(this.consentService.consentsStatus),
-        concatMap((consent) =>
+        concatMap((consents) =>
           this.orderBrokerService.updateOrder(
             this.draftOrder.orderId,
             {
-              consent: consent?.filter(
+              consent: consents?.filter(
                 (consent) =>
                   consent.consentTemplateType === ConsentType.PPANDTOS ||
                   consent.consentTemplateType === ConsentType.NEXTSELL
               ),
             },
-            { allowAsync: this.appData.useAsync }
+            { orderLineResponseViewType: true }
           )
         ),
+        concatMap(() => {
+          let calculateCostRequest = of(void 0);
```

Comment:
I recommend this being in its own private method.

Reply:
I guess I can do that... be easier to rip out later